app/services/mapping/goodsshipment/ConsigneeBuilder.scala

```diff
@@ -16,11 +16,22 @@
 
 package services.mapping.goodsshipment
 import forms.declaration.{ConsigneeDetails, EntityDetails}
+import javax.inject.Inject
 import services.Countries.allCountries
+import services.mapping.goodsshipment.ConsigneeBuilder.createConsignee
 import uk.gov.hmrc.http.cache.client.CacheMap
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment
 import wco.datamodel.wco.declaration_ds.dms._2._
 
+class ConsigneeBuilder @Inject()() {
+
+  def buildThenAdd(maybeConsigneeDetails: Option[ConsigneeDetails], goodsShipment: GoodsShipment) =
```

Comment:
What's the point in passing in an Option to a builder? Could always just not call the builder if the argument is `None`

I think that's how the other builders work

Reply:
have fixed this to not be an option